### PR TITLE
Clarify that cubes do not need to be solved at the start of FMC

### PR DESCRIPTION
**Pull request by ohexter**

### CODE REVIEW

File: wca-guidelines.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # <wca-title>WCA Guidelines
 
-<version>Version: January 1, 2024
+<version>Version: January 1, 2025
```
Comment (Nevseros) [inline wca-guidelines.md]:
```suggestion
<version>Version: January 1, 2024
```

File: wca-regulations.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # <wca-title>WCA Regulations
 
-<version>Version: January 1, 2024
+<version>Version: January 1, 2025
```
Comment (Nevseros) [inline wca-regulations.md]:
```suggestion
<version>Version: January 1, 2024
```

**Comment by OJCubing** [comment]
Should be 3x3x3 Cube

File: wca-guidelines.md [hunk 3]
```diff
@@ -233,6 +233,7 @@ To be more informative, each Guideline is classified using one of the following
 - E2c4+) [CLARIFICATION] Examples of symbols or combinations of symbols that are not permitted in solutions for 3x3x3 Fewest Moves: M, e, r, Y (uppercase), T, U1, Ui, U3, U-, R++, R2', R'2, L'w, 1R, 2U, 2f, 2-3r, 1Rw, 3Rw, 2-3Rw, rw, Rr, *, →, ⮐, /, ., ?, [r], [r u], [F], &lt;f&gt;, [f]', [f]2, [x], (x), (L), 2(Rw), (U)2, (U)', (R U)', (R U R'), [R, U], [R: U].
 - E2e+) [CLARIFICATION] Competitors must not derive solutions from any part of the scramble sequence, and solutions should not share significant parts with the inverse scramble sequence.
 - E2e++) [EXAMPLE] Example of solutions that should result in a disqualification of the attempt (DNF): solutions beginning with the same 4 or more moves as the inverse scramble sequence.
+- E3b+) [CLARIFICATION] It is not necessary for any 3x3 Cubes to be solved at the start of the attempt.
```
Comment (ohexter) [inline wca-guidelines.md]:
```suggestion
- E3b+) [CLARIFICATION] It is not necessary for any 3x3X3 Cubes to be solved at the start of the attempt. 
```

File: wca-guidelines.md [hunk 4]
```diff
@@ -233,6 +233,7 @@ To be more informative, each Guideline is classified using one of the following
 - E2c4+) [CLARIFICATION] Examples of symbols or combinations of symbols that are not permitted in solutions for 3x3x3 Fewest Moves: M, e, r, Y (uppercase), T, U1, Ui, U3, U-, R++, R2', R'2, L'w, 1R, 2U, 2f, 2-3r, 1Rw, 3Rw, 2-3Rw, rw, Rr, *, →, ⮐, /, ., ?, [r], [r u], [F], &lt;f&gt;, [f]', [f]2, [x], (x), (L), 2(Rw), (U)2, (U)', (R U)', (R U R'), [R, U], [R: U].
 - E2e+) [CLARIFICATION] Competitors must not derive solutions from any part of the scramble sequence, and solutions should not share significant parts with the inverse scramble sequence.
 - E2e++) [EXAMPLE] Example of solutions that should result in a disqualification of the attempt (DNF): solutions beginning with the same 4 or more moves as the inverse scramble sequence.
+- E3b+) [CLARIFICATION] It is not necessary for any 3x3X3 Cubes to be solved at the start of the attempt.
```
Comment (Nevseros) [inline wca-guidelines.md]:
```suggestion
- E3b+) [CLARIFICATION] It is not necessary for any 3x3x3 Cubes to be solved at the start of the attempt. 
```

**Comment by CarterKoala** [comment]
LGTM

File: wca-guidelines.md [hunk 5]
```diff
@@ -233,6 +233,7 @@ To be more informative, each Guideline is classified using one of the following
 - E2c4+) [CLARIFICATION] Examples of symbols or combinations of symbols that are not permitted in solutions for 3x3x3 Fewest Moves: M, e, r, Y (uppercase), T, U1, Ui, U3, U-, R++, R2', R'2, L'w, 1R, 2U, 2f, 2-3r, 1Rw, 3Rw, 2-3Rw, rw, Rr, *, →, ⮐, /, ., ?, [r], [r u], [F], &lt;f&gt;, [f]', [f]2, [x], (x), (L), 2(Rw), (U)2, (U)', (R U)', (R U R'), [R, U], [R: U].
 - E2e+) [CLARIFICATION] Competitors must not derive solutions from any part of the scramble sequence, and solutions should not share significant parts with the inverse scramble sequence.
 - E2e++) [EXAMPLE] Example of solutions that should result in a disqualification of the attempt (DNF): solutions beginning with the same 4 or more moves as the inverse scramble sequence.
+- E3b+) [CLARIFICATION] It is not necessary for any 3x3x3 Cubes to be solved at the start of the attempt.
```
Comment (Nanush7) [inline wca-guidelines.md]:
```suggestion
- E3b+) [CLARIFICATION] Puzzles do not need to be solved at the start of the attempt. 
```
Alternative (simpler?) option.

Comment (ohexter) [reply wca-guidelines.md]: Using 3x3x3 Cubes is consistent with the parent regulation, but I don't feel particularly strongly either way.

**Pull request merged**